CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Views/CameraPane.swift

```diff
@@ -103,15 +105,60 @@ final class CameraPane: UIView {
     }
 
     func setupAuthorization(isHidden: Bool) {
+        let giniConfiguration = GiniConfiguration.shared
         self.isHidden = isHidden
         captureButton.isHidden = isHidden
         flashButton.isHidden = isHidden
         if cameraTitleLabel != nil {
             cameraTitleLabel.isHidden = isHidden
         }
-        fileUploadButton.isHidden = isHidden
+        if giniConfiguration.fileImportSupportedTypes != .none {
+            fileUploadButton.isHidden = isHidden
+        }
         if thumbnailView.thumbnailImageView.image != nil {
             thumbnailView.isHidden = isHidden
         }
+        updateThumbnailConstraint()
+    }
+
+    private func numberOfVisibleButtons() -> Int {
+        var number = 2
```

Comment:
[Suggestion] Maybe it is rocket science, but: `[flashButton.isHidden, fileUploadButton.isHidden].filter({$0}).count`